modules/Hoot/config/apiConfig.js

```diff
@@ -5,13 +5,13 @@
  *******************************************************************************************************/
 
 export const apiConfig = {
-  host: "http://54.165.118.78",
-  port: "8080",
-  path: "hoot-services",
-  translationServerPort: "8094",
-  mapnikServerPort: "8000",
-  mergeServerPort: "8096",
-  queryInterval: 2000
+    host: 'http://localhost',
+    port: '8080',
+    path: 'hoot-services',
+    translationServerPort: '8094',
+    mapnikServerPort: '8000',
+    mergeServerPort: '8096',
+    queryInterval: 2000
```

Comment:
Will this need updating per deployed environment?

Reply:
AFAIK, this should be consistent with the ports in production

Reply:
Could this use `window.location.host` and `window.location.port` instead?

Reply:
Unless you're talking about the UI, that wouldn't work because the translation, mapnik, and merge servers run on different port from the UI.

Reply:
Also, the api config is bundled alongside the rest of the code so we'd have to update ports during the bundling process. This can be done with node environment variables

Reply:
Just for the protocol, hostname, port to hit the UI.